src/main/docs/guide/views/rocker.adoc

```diff
@@ -0,0 +1,89 @@
+Micronaut includes api:views.rocker.RockerEngine[] which uses
+the https://github.com/fizzed/rocker[Rocker] Java-based template engine.
+
+In addition to the <<views, Views>> dependency, add the following dependency on your classpath.
```

Comment:
It's no longer recommended to use the `views` dependency, but rather use the `rocker-views` dependency in this example. That means users no longer have to add the template specific dependency in addition.

Reply:
`views` dependency removed.